.github/ISSUE_TEMPLATE/first_rc_checklist.md

```diff
@@ -0,0 +1,30 @@
+---
+name: First Release Candidate Checklist (maintainer only)
+about: Checklist template for the first release of every series
+title: llvmlite X.Y.Zrc1 Checklist (FIXME)
+labels: task
+
+---
+
+
+## llvmlite X.Y.Z
+
+* [ ] Merge to `main`.
+    - [ ] "remaining Pull-Requests from milestone".
+* [ ] Merge change log changes.
+    - [ ] "PR with changelog entries".
+* [ ] Create X.Y release branch.
+* [ ] Build conda packages on buildfarm (check "upload").
+* [ ] Verify packages uploaded to Anaconda Cloud and move to `numba/label/main`.
+* [ ] Build wheels (`$PYTHON_VERSIONS`) on the buildfarm.
+* [ ] Upload wheels and sdist to PyPI and verify arrival. (upload from `ci_artifacts`).
+* [ ] Initialize and verify ReadTheDocs build.
+* [ ] Send RC announcement email / post announcement to discourse group.
+* [ ] Post link to Twitter.
```

Comment:
This section differs to what is in the equivalent in Numba and might need an update.

Reply:
What do you mean, specifically?